libnavigation-ui/src/main/res/layout-land/instruction_content_layout.xml

```diff
@@ -81,6 +85,8 @@
             android:maxLines="1"
             android:textSize="22sp"
             android:textStyle="bold"
+            android:textDirection="rtl"
```

Comment:
I think I made a mistake here. This should be `locale` instead of `rtl`. Can you please confirm? We probably might wanna check if we need these two properties and what happens if we remove them.